pkg/dns/util/util.go

```diff
@@ -42,14 +46,51 @@ const (
 // ExtractIP turns a standard PTR reverse record lookup name
 // into an IP address
 func ExtractIP(reverseName string) (string, bool) {
-	if !strings.HasSuffix(reverseName, ArpaSuffix) {
-		return "", false
+	if strings.HasSuffix(reverseName, ArpaSuffix) {
```

Comment:
Can we change the signature to return error so we can log why something was not parsed?

Reply:
done